### PR TITLE
feat(web): remove export in pst format

### PR DESCRIPTION
**Pull request by antonymarion**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master` 
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | MANAGER-13636
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [ ] ~~Only FR translations have been updated~~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~

### CODE REVIEW

**Review by anooparveti — COMMENTED** [review]
**question:** If the template `account-export-as-pst` is not used anywhere else, can you remove that as well?

**Review by qpavy — APPROVED** [review]

**Review by qpavy — APPROVED** [review]

File: packages/manager/modules/exchange/src/account/account.controller.js [hunk 1]
```diff
@@ -96,6 +96,7 @@ export default class ExchangeAccountHomeController {
       this.refreshList(),
     );
 
+    this.isPrivate = this.productId.includes('private-ls');
```
Comment (anooparveti) [inline packages/manager/modules/exchange/src/account/account.controller.js:99]:
```suggestion
    this.isPrivateExchange = this.productId.includes('private-ls');
```

File: packages/manager/modules/exchange/src/account/account.controller.js [hunk 2]
```diff
@@ -96,6 +96,7 @@ export default class ExchangeAccountHomeController {
       this.refreshList(),
     );
 
+    this.isPrivateExchange = this.productId.includes('private-ls');
```
Comment (ghyenne) [inline packages/manager/modules/exchange/src/account/account.controller.js]:
```suggestion
    this.isPrivateExchange = this.productId.includes('private-');
```

Comment (ghyenne) [reply packages/manager/modules/exchange/src/account/account.controller.js]: I think no need to add think check
You can see in packages/manager/modules/exchange/src/dashboard/Exchange.class.js normaly this check is already done. For information private-xxxxxxx is "dedicated" offer

**Review by ghyenne — CHANGES_REQUESTED** [review]

File: packages/manager/modules/exchange/src/account/account.controller.js [hunk 3]
```diff
@@ -96,6 +96,7 @@ export default class ExchangeAccountHomeController {
       this.refreshList(),
     );
 
+    this.isPrivateExchange = this.productId.includes('private');
```
Comment (ghyenne) [inline packages/manager/modules/exchange/src/account/account.controller.js:99]: I think no need to add think check
You can see in packages/manager/modules/exchange/src/dashboard/Exchange.class.js normaly this check is already done. For information private-xxxxxxx is "dedicated" offer

Comment (antonymarion) [reply packages/manager/modules/exchange/src/account/account.controller.js:99]: I will check this and change thks

**Review by ghyenne — APPROVED** [review]

**Review by qpavy — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**